### PR TITLE
chore: reformat doc

### PR DESCRIPTION
**Pull request by liuxiaocs7**

### CODE REVIEW

File: hugegraph-ml/README.md
Comment (imbajin) [inline hugegraph-ml/README.md:1]: emm, unexpected header?

Comment (liuxiaocs7) [reply hugegraph-ml/README.md:1]: Missing header when using gitkeep, so add  temporarily

image

Comment (liuxiaocs7) [reply hugegraph-ml/README.md:1]: removed

**Review by simon824 — APPROVED** [review]
LGTM

**Pull request merged**